modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,41 @@
+//
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+Network Observability requires a substantial amount of resources. There are settings to consider that can help you get an optimal setup without compromising on the observability.
+
+The following settings can help you manage resources and performance:
```

Comment:
We might want to call out eBPF sampling rate here, or link to somewhere where it is explained

Reply:
Agreee, this is covered elsewhere but its not really linkable; however, I think its OK to mention it in full text again here. The other place its discussed is the Viewing FlowCollector resource topic in the Configuration guidance and its mentioned as a resource management tool there, so its good to mention here as well for sure.